DateTimePicker.js

```diff
@@ -60,22 +61,22 @@ const styles = StyleSheet.create({
 
 export default class DateTimePicker extends Component {
     static propTypes = {
-        date: React.PropTypes.oneOfType([
-            React.PropTypes.date,
-            React.PropTypes.string,
+        date: PropTypes.oneOfType([
+            PropTypes.date,
+            PropTypes.string,
         ]),
-        iosDoneButtonText: React.PropTypes.string,
-        iosDoneButtonStyle: Button.propTypes.containerStyle,
-        iosClosePickerButtonTextContainerStyle: Button.propTypes.textContainerStyle,
-        iosClosePickerButtonTextStyle: Button.propTypes.textStyle,
-        label: React.PropTypes.string,
-        mode: React.PropTypes.oneOf([
+        iosDoneButtonText: PropTypes.string,
+        iosDoneButtonStyle: PropTypes.containerStyle,
```

Comment:
I'm not sure this would work, I doubt the React `PropTypes` have a `containerStyle` type.

I think the `Button.propTypes` should stay that way, as they are using `react-native-osd-simple-button`'s propTypes